ui/src/ethlance/ui/page/job_contract/subscriptions.cljs

```diff
@@ -1,5 +1,16 @@
 (ns ethlance.ui.page.job-contract.subscriptions
   (:require [ethlance.ui.page.job-contract.events :as job-contract.events]
-            [ethlance.ui.subscription.utils :as subscription.utils]))
+            [ethlance.ui.subscription.utils :as subscription.utils]
+            [re-frame.core :as re]))
 
 (def create-get-handler #(subscription.utils/create-get-handler job-contract.events/state-key %))
+
+; (re/reg-sub :page.job-contract/feedback-rating (create-get-handler :feedback-rating))
+(re/reg-sub
+  :page.job-contract/feedback-rating
+  (fn [db _]
+    ; Debugging: This works and gets run when user clicks c-rating stars
+    (println ":page.job-contract/feedback-rating subscription updated" (get-in db [job-contract.events/state-key :feedback-rating]))
```

Comment:
When clicking the star (rating) this log is produced

![Screenshot from 2023-03-22 10-38-00](https://user-images.githubusercontent.com/674621/226975327-786657e5-c1de-4eb4-b4a0-7df4e2e69dd3.png)